tools/draw-importances.py

```diff
@@ -1,55 +1,180 @@
 #!/usr/bin/env python3
 import struct
 import sys
+import os
 from os.path import splitext
+import datetime
 import numpy as np
 from PIL import Image, ImageDraw
 from matplotlib import pyplot as plt
+import click
```

Comment:
Is this module bundled with default Python 3 installs?

Reply:
Nop this is not bundled by default in python3, 
We require to install, either we can add a README.md in the [src/tools](https://github.com/xiph/rav1e/tree/master/tools), explaining the scripts in a short and simple way, 
or we can leave it as it is, since it is mostly used for internal evaluation, 
EIther way I am fine :)

Reply:
Do we use click anywhere else in the repo? If not maybe it'd be better to stick to argparse?

Reply:
Actually since we already use a few packages like numpy and matplotlib it should be fine. Took me a bit to realize, these have just become so standard to me that I don't even think about them. 😅